BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -307,6 +308,7 @@ public final class ConvivaAnalytics: NSObject {
         sessionKey = NO_SESSION_KEY
         contentMetadataBuilder.reset()
         logger.debugLog(message: "Session ended")
+        isPlaybackStarted = false
```

Comment:
Just a nit. Better to announce "Session ended" as last statement, after cleaning everything:-)

Reply:
That is a valuable nit. Updated.